interfaces/acados_template/acados_template/c_templates_tera/acados_solver.in.c

```diff
@@ -2471,6 +2481,16 @@ static void {{ model.name }}_acados_create_set_opts({{ model.name }}_solver_caps
     free(sim_method_jac_reuse);
   {%- endif %}
 
+{%- if solver_options.integrator_type == "ERK" %}
```

Comment:
I would remove the if here and add sanity checks that `sens_forw_p` can only be True if ERK is used in `make_consistent` in the high level interfaces.